ota-plus-web/app/reactapp/src/components/profile/ProfileOrganization/index.jsx

```diff
@@ -121,8 +130,22 @@ class ProfileOrganization extends Component {
     this.setState({ menuEditShowMenu: false });
   };
 
+  openRemovalModal = () => {
+    this.setState({ isRemovalModalOpen: true });
+  }
+
+  openSelfRemovalModal = () => {
+    this.setState({ isSelfRemovalModalOpen: true });
+  }
+
+  closeRemovalModal = () => {
+    this.setState({ isRemovalModalOpen: false, isSelfRemovalModalOpen: false });
+  }
+
   render() {
     const {
+      isRemovalModalOpen,
```

Comment:
Please use string (removalModalOpenType, default is undefined) and string enum with two values (REMOVAL_MODAL and SELD_REMOVAL_MODAL) instead of those two booleans. Then create function to return object with translated strings and use it in render function instead of duplicating WarningModal usage.